CHANGELOG.md

```diff
@@ -1,13 +1,124 @@
 # Changelog
-All notable changes to this project prior to release **0.22.0** are documented in this file. Future
-changelog information can be found in each release's git tag and can be viewed with `git tag -ln100 "v*"`.
-Changelog info is also documented on the [GitHub releases](https://github.com/bitcoindevkit/bdk/releases)
-page. See [DEVELOPMENT_CYCLE.md](DEVELOPMENT_CYCLE.md) for more details.
+
+All notable changes to this project can be found here and in each release's git tag and can be viewed with `git tag -ln100 "v*"`. See also [DEVELOPMENT_CYCLE.md](DEVELOPMENT_CYCLE.md) for more details.
```

Comment:
Do you think it would be useful to add a line here explaining that contributors shouldn't modify this file?

Reply:
Fixed.